dipy/reconst/recspeed.pyx

```diff
@@ -54,10 +54,11 @@ cdef void splitoffset(float *offset, size_t *index, size_t shape) nogil:
 def trilinear_interp(cnp.ndarray[cnp.float32_t, ndim=4, mode='strided'] data,
                      cnp.ndarray[cnp.float_t, ndim=1, mode='strided'] index,
                      cnp.ndarray[cnp.float_t, ndim=1, mode='c'] voxel_size):
-    """Interpolates data at index
+    """ Interpolates vector from 4D `data` at 3D point given by `index`
```

Comment:
Maybe put a newline in there, instead of that space (at the very beginning of this line). If you feel like it, you can put newlines at the beginning of many docstrings in these files...